ex03.ex

```diff
@@ -77,8 +79,11 @@ defmodule Ex03 do
 
   """
 
-  def list_contains . .. "your code"
+  def list_contains([h | _], val) when (h == val), do: true
```

Comment:
## -1

Why use a test when you can pattern match?

~~~ elixir
def list_contains([ val | _ ], val), do: true
~~~